notebooks/charactercodex_llamafile.ipynb

```diff
@@ -0,0 +1,2607 @@
+{
```

Comment:
<p>I would add 'Llamafile' to somewhere in the title.</p><p>This makes me think that we should start figuring out whether we should organize these cookbooks somehow :D</p><p>It'd difficult to filter</p><p> </p><p>In this first paragraph, could you in a few words write what we will be building by the end of this cookbook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done.</p><p> </p><p>Agree that we should probably start organizing the cookbook...</p>